handlers.rb

```diff
@@ -0,0 +1,104 @@
+require_relative 'person'
+require_relative 'teacher'
+require_relative 'student'
+require_relative 'book'
+require_relative 'rental'
+
+module Handlers
+  def list_books
+    @books.each { |book| puts book }
+  end
+
+  def list_people
+    @people.each { |person| puts person }
+  end
+
+  def create_person
+    print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+    option = gets.chomp
```

Comment:
all requests for input and input validation should've been placed in the `main.rb`
please refer to [Best practices](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid)

- [ ] Kindly refactor

Reply:
Pay special attention to the "Single-responsiblity Principle".

    Make sure that the role of main.rb is only to start your code and ask the user for the initial choice of options (or to quit).
    Extract some pieces of logic into separate class(es).

Reply:
@usmansbk this inputs can be hndled by the individual classes after the user makes a choice from the initial inputs in the `main.rb` file, as with the rest 🙂